noxfile.py

```diff
@@ -298,6 +298,8 @@ def test_plugins(session, install_cmd):
         session.chdir(os.path.join(BASE, "plugins", plugin.path))
         if plugin.name == "hydra-ray-launcher":
             session.run("pytest", "-Werror", ".", silent=False)
+            # Install ray launcher first causes cython issues with ax-sweeper in nox
+            session.run("pip", "uninstall", "-y", "hydra-ray-launcher")
```

Comment:
We need to dig into this incompatibility.
we can't just hack the noxfile for this.

Are the ax_sweeper and ray launcher incompatible?

This seems odd because Ray actually supports Ax (through Ray Tune).

Reply:
This is the stack trace when first installing ray and then ax
https://www.internalfb.com/intern/paste/P133414964/

This only happens when running the tests in nox mode (I cannot reproduce the test failures in my local conda env or on the circleCI instances by running ```pytest .``` directly. )

I've spent quite some time on debugging but haven't located the issue. Let's chat in person.